lib/screens/profile/faq.dart

```diff
@@ -14,12 +29,25 @@ class FaqScreen extends StatelessWidget {
         backgroundColor: MyColors.backgroundColor,
       ),
       body: Container(
-        //color: MyColors.backgroundColor,
         padding: EdgeInsets.all(9),
-        child: SingleChildScrollView(
-          child: Column(
-            children: List.generate(7, (index) => const Faqtiles()),
-          ),
+        child: FutureBuilder<List<Faqs>>(
+          future: faqListFuture,
+          builder: (context, snapshot) {
+            if (snapshot.connectionState == ConnectionState.waiting) {
+              return Center(child: CircularProgressIndicator());
+            } else if (snapshot.hasError) {
+              return Center(child: Text('Error: ${snapshot.error}'));
+            } else if (!snapshot.hasData || snapshot.data!.isEmpty) {
+              return Center(child: Text('No FAQs available.'));
+            } else {
+              List<Faqs> faqsList = snapshot.data!;
+              return SingleChildScrollView(
+                child: Column(
+                  children: faqsList.map((faq) => Faqtiles(faq: faq)).toList(),
```

Comment:
No need to use the variabla faqsList
use snapshot.data directly in column widget